DuggaSys/diagram.php

```diff
@@ -145,7 +146,12 @@
     </div> 
     <div id="fab" onclick="fab_action();" onmousedown='mdown(event)'>+</div> <!-- Big (+) button -->
     <div id="options-pane" class="hide-options-pane" onmousedown='mdown(event)'> <!-- Yellow menu on right side of screen -->
-        <div id="options-pane-button" onclick="fab_action();"><span id='optmarker'>&#9660;Options</span></div>
+        <div id="options-pane-button" onclick="fab_action();"><span id='optmarker'>&#9660;Options</span>
+        <span class="toolTipText"><b>Show Option Panel</b><br>
+                    <p>Enable/disable the Option Panel</p><br>
+                    <p id="tooltip-OPTIONS" class="key_tooltip">Keybinding:</p>
+                </span>
+        </div>
```

Comment:
Indentation is weird here, span is child of div, but then span should be indented one step. Also p elements seem to be indented too much. The closing span tag at line 153 should also be directly under the opening tag of span (indentation wise)